portfolio-project-react/src/components/SignUp/SignUp.jsx

```diff
@@ -58,47 +108,56 @@ const SignUp = () => {
         >
           <h2>{SIGNUP_TITLE}</h2>
           <input
+            value={name}
             placeholder={PLACEHOLDER_NAME}
             style={{
-              borderColor: nameError && isValidated ? "red" : "transparent",
+              borderColor:
+                nameError && isValidated
+                  ?  COLOR_RED 
+                  :  COLOR_TRANSPARENT ,
```

Comment:
Instead of using conditional classes in style prop. Please add the CLSX npm package and handle classname conditionally.